tools/exports.js

```diff
@@ -1,15 +1,4 @@
-exports["Compressor"] = Compressor;
 exports["Dictionary"] = Dictionary;
-exports["JS_Parse_Error"] = JS_Parse_Error;
```

Comment:
So instead of throwing, return with an exception instead?

I suppose it's not a bad idea, seeing that people don't like to wrap `try-catch` all the time if they can avoid it. The quick patch I had in mind can't return anything else than the `Error` instance though, so no point wrapping it within another object as the return value?